react/src/component/PerfTestDashboard.jsx

```diff
@@ -39,21 +39,35 @@ const androidMemoryOptions = [
     { value: 'unknownPss', label: 'unknownPss', color: '#8884d8' },
     { value: 'unknownRss', label: 'unknownRss', color: '#8884d8' }
 ]
+const windowsMemoryOptions = [
+    { value: 'nonpagedSystemMemorySize64', label: 'nonpagedSystemMemorySize64', color: '#007FFF' },
+    { value: 'pagedMemorySize64', label: 'pagedMemorySize64', color: '#FFA500' },
+    { value: 'pagedSystemMemorySize64', label: 'pagedSystemMemorySize64', color: '#8B8970' },
+    { value: 'peakPagedMemorySize64', label: 'peakPagedMemorySize64', color: '#800000' },
+    { value: 'peakWorkingSet64', label: 'peakWorkingSet64', color: '#808000' },
+    { value: 'privateMemorySize64', label: 'privateMemorySize64', color: '#4B0080' },
+    { value: 'workingSet64', label: 'workingSet64', color: '#8884d8' },
+    { value: 'peakVirtualMemorySize64', label: 'peakVirtualMemorySize64', color: '#FFCC00' },
+]
 
 export default class PerfTestDashboard extends React.Component {
     state = {
         perfTestResult: this.props.perfTestResult,
         memoryInfo: undefined,
         batteryInfo: undefined,
+        windowsMemoryInfo: undefined,
         selectedAndroidBatteryOptions: androidBatteryOptions.slice(0, 4),
         selectedAndroidMemoryOptions: androidMemoryOptions.slice(0, 10),
+        selectedWindowsMemoryOptions: windowsMemoryOptions.slice(0, 7)
     };
 
     render() {
         const memoryInfo = this.state.memoryInfo;
         const memoryMetrics = [];
         const batteryInfo = this.state.batteryInfo;
         const batteryMetrics = [];
```

Comment:
We could also add `Android` prefix to these variables

Reply:
Sure.
Android memory report also works well.
![image](https://user-images.githubusercontent.com/10386842/224949817-1539b484-b4ee-4039-9a0a-8bd2ce4c78cd.png)